acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature

```diff
@@ -14,76 +13,856 @@ Feature: Open ongoing order
+    When method GET
+    And match $.totalRecords == 2
+
+  Scenario: Change instance connection for the order to some member tenant, where the shadow instance is located.
+  Verify that associated holdings and items moved to the same tenant
+
+    ## 1. Create Order and orderLine
+    # 1.1 Create orders
+    * def v = call createOrder { id: '#(orderId)', vendor: '#(centralVendorId)', orderType: 'One-Time', ongoing: null }
+
+    # 1.2 Create order lines with location in member and newly created holdingId in central tenant
+    Given path 'orders/order-lines'
+
+    * def orderLine = read('classpath:samples/consortia/orderLines/multi-tenant-order-line.json')
+    * set orderLine.id = poLineId
+    * set orderLine.purchaseOrderId = orderId
+    * set orderLine.cost.listUnitPrice = 100
+    * set orderLine.fundDistribution[0].fundId = fundId
+    * set orderLine.locations[2].tenantId = universityTenant
+    * set orderLine.locations[2].locationId = universityLocationsId
+    * set orderLine.titleOrPackage = 'title 1'
+
+    And header x-okapi-tenant = centralTenant
+    And request orderLine
+    When method POST
+    Then status 201
+
+    ## 2. Open order
+
+    Given path 'orders/composite-orders', orderId
+    When method GET
+    And header x-okapi-tenant = centralTenant
+    Then status 200
+
+    * def orderResponse = $
+    * set orderResponse.workflowStatus = "Open"
+
+    Given path 'orders/composite-orders', orderId
+    And header x-okapi-tenant = centralTenant
+    And request orderResponse
+    When method PUT
+    Then status 204
+
+
+    ## 3. Verify Instance, Holding, Item in 'centralTenant'
+
+    # 3.1 Check the order line have an instanceId 'centralInstanceId1'
+    Given path 'orders/order-lines', poLineId
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    Then status 200
+    And match $.instanceId == '#present'
+    * def centralPoLinInstanceId1 = $.instanceId
+
+    # 3.2 Check instances in 'centralTenant'
+    Given path 'inventory/instances', centralPoLinInstanceId1
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    Then status 200
+
+    # 3.3 Check holdings with location in 'centralTenat'
+    Given path 'holdings-storage/holdings'
+    And param query = 'instanceId==' + centralPoLinInstanceId1
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    Then status 200
+    And match $.totalRecords == 2
+    * def centralPoLineHoldingId1 = response.holdingsRecords[0].id
+    * def centralPoLineHoldingId2 = response.holdingsRecords[1].id
+
+    # 3.4 Check items in 'centralTenant'
+    Given path 'inventory/items'
+    And param query = 'purchaseOrderLineIdentifier==' + poLineId
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    And match $.totalRecords == 2
+    * def centralPoLineItemId1 = response.items[0].id
+    * def centralPoLineItemId2 = response.items[1].id
+
+    ## 4. Verify Shadow Instance, Holding, Item in 'centralTenant'
```

Comment:
comment should be updated to memberTenant

Reply:
Thanks, updated